test/jdk/javax/swing/JMenuItem/8158566/CloseOnMouseClickPropertyTest.java

```diff
@@ -40,6 +44,7 @@
  * @key headful
  * @bug 8158566 8160879 8160977 8158566
  * @summary Provide a Swing property which modifies MenuItemUI behaviour
+ * @run main/othervm/timeout=600 CloseOnMouseClickPropertyTest
```

Comment:
More than 2 minutes you wrote.
I just ran it on mac and it was about 43 seconds
java CloseOnMouseClickPropertyTest  7.21s user 2.21s system 22% cpu 42.608 total

But 600 seconds is 10 minutes
And when we run these tests on our CI system to be conservative we tell jtreg to use 2X the specified timeout.  If it needs 20 minutes I don't think it is ever going to finish.

I think a timeout specified here of maybe 240 is sufficient.

Reply:
I was thinking specifying 600secs doesn;t mean the test will take all of it if it can finish early, it is definitely taking more time in this ubuntu systems..As I told, running "part1" twice was working ok for this test with timeout=600, so test does finish within this timeout..I have posted 2 jobs in JBS with timeout=600 where this test passed which shows the test finished once in "elapsed time (seconds): 690.032" and another in "elapsed time (seconds): 717.237"

Moreso, seems like test fails with suggested timeout=240

Reply:
Reduce timeout further where it works..

Reply:
Reduced timeout further..